lib/Bio/EnsEMBL/DataCheck/index.json

```diff
@@ -191,6 +191,13 @@
       "name" : "PhenotypeDescription",
       "package_name" : "Bio::EnsEMBL::DataCheck::Checks::PhenotypeDescription"
    },
+   "RegulatoryFeatureIsActive" : {
+      "datacheck_type" : "critical",
+      "description" : "Check that every regulatory_feature of the current Regulatory Build has a valid activity value in at least one epigenome.",
+      "groups" : [],
```

Comment:
The groups in the index don't match those in the datacheck (which is why travis is failing). Presumably you added the groups after you had run the creation script. I belatedly added a note to the docs that in such cases you need to run create_index.pl (https://www.ebi.ac.uk/seqdb/confluence/display/GTI/Everything+you+wanted+to+know+about+datachecks%2C+but+were+too+afraid+to+ask#Everythingyouwantedtoknowaboutdatachecks,butweretooafraidtoask-Creatingadatachecktemplate)

But it's quite annoying to have to remember to do this, when it seems like it could be automated. I've never used git hooks, but I'll look into whether they could be useful here.

Reply:
Arg you are right. I've now run the index script.